### ISSUE TITLE
Amazon Lambda TestTool 5.0 Starting Paused

### ISSUE THREAD

**Issue by InzeNL** [aws/aws-lambda-dotnet#893]
## The Question
I will reference this issue, because it describes a similar issue. However, it is for a different .NET platform, so I decided to create a separate issue. [https://github.com/aws/aws-lambda-dotnet/issues/804](https://github.com/aws/aws-lambda-dotnet/issues/804)

I am currently working on developing a Lambda function in C# using net5.0. I have set up the project using Visual Studio using the AWS Toolkit: AWS Lambda Project (.NET Core - C#) > Custom Runtime Function.
I have also installed the Lambda TestTool 5.0 to be able to use the Mock Lambda Test Tool. For Visual Studio, I have added this part to the `Properties\launchSettings.json`, as mentioned in the `Readme.md` generated with the project
```json
{
  "profiles": {
    "Mock Lambda Test Tool": {
      "commandName": "Executable",
      "commandLineArgs": "--port 5050",
      "workingDirectory": ".\\bin\\$(Configuration)\\net5.0",
      "executablePath": "%USERPROFILE%\\.dotnet\\tools\\dotnet-lambda-test-tool-5.0.exe"
    }
  }
}
```
When I select the project and click "Mock Lambda Test Tool," a page does open up, and I see the test tool. However, there is a white opaque overlay with the following message:
> AWS .NET Core 5.0 Mock Lambda Test Tool has paused. This can happen for the following reasons.
>
> The attached debugger is currently on a breakpoint. The test tool will resume when execution has continued in the debugger.
> The debugger or test tool process has stopped. If so, this window can be closed.

I have no breakpoints in my code, and the command prompt is still open.

I have also tried using the test tool from the command line, in the project root directory, as follows:
```
dotnet lambda-test-tool-5.0
```

And from the `%userprofile%\.dotnet\tools` directory, as follows:
```
dotnet-lambda-test-tool-5.0.exe --path "A:\Path\To\My\Project"
```

What am I doing wrong?

### Environment
* **Build Version:**
  * *amazon.lambda.tools* 5.1.4
  * *amazon.lambda.testtool-5.0* 0.11.3
  * *AWS Toolkit for Visual Studio 2017 and 2019* 1.21.2.0
* **OS Info:**
  * Windows 10 Home 64-bit (10.0, Build 19042)
* **Build Environment:**
  * Visual Studio 2019
  * Terminal `dotnet`
* **Targeted .NET Platform:**
  * net5.0

---

This is a :question: general question

**Comment by ashishdhingra** [comment]
Hi @InzeNL,

Good morning.

Thanks for posting guidance question. Somehow, I'm unable to reproduce the issue. As mentioned in the https://github.com/aws/aws-lambda-dotnet/issues/804#issuecomment-775384576, if I stop debugging, then I see that white opaque overlay with the specified message. 
![LambdaTestTool_Debug](https://user-images.githubusercontent.com/67916761/126370037-0df013b3-d8fd-4e3d-944e-a5bea64a1b7a.png)
![Lambda](https://user-images.githubusercontent.com/67916761/126370053-53236e10-158e-47c9-b05e-4c7d5c7f7e18.png)


Try closing the browser windows and also see if Visual Studio break on all exceptions if turned off.

Thanks,
Ashish

**Comment by InzeNL** [comment]
Looking at your screenshots, I notice you have Visual Studio open a custom Chrome browser (recognizing it mainly by this)
![image](https://user-images.githubusercontent.com/2505862/126393932-1236f9d0-64c8-4826-8c0e-0bfdeb2ab68b.png)

I don't have this anymore. When I started on my project, I used to get this custom browser while launching my IIS Server, too. However, since then I've gone through a PC reset, and since then it opens the URL in my regular browser. I'm thinking this is also what is causing the issue of the Mock tool not working. The browser link does not seem to work properly, when I close the browser, the Mock tool keeps running.

I will do some investigating myself as to how to fix this. And will report back when I have found a solution.
